ecommerce-demo/workspaces/childDemo_clothing/workspaceDataFilterSettings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-clothing",
+        "attributes": {
+          "title": "ecommerce merchant clothing",
+          "filterValues": [
+            "merchant__clothing"
+          ]
+        },
+        "relationships": {
+          "workspaceDataFilter": {
+            "data": {
+              "id": "WDF__CLIENT_ID",
+              "type": "workspaceDataFilter"
+            }
+          }
+        }
+      }
+    },
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-clothing-cat",
```

Comment:
Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_clothing_cat",
```